packages/updating-element/src/css-tag.ts

```diff
@@ -105,3 +107,38 @@ export const css = (
   }
   return result;
 };
+
+/**
+ * Applies the given styles to a `shadowRoot`. When Shadow DOM is
+ * available but `adoptedStyleSheets` is not, styles are appended to the the
+ * `shadowRoot` to [mimic spec behavior](https://wicg.github.io/construct-stylesheets/#using-constructed-stylesheets).
```

Comment:
I think we should mention that rendered DOM should go _before_ emulated adopted styles to match the spec for ordering between `<style>` tags and adopted styles.

Reply:
Done.